contribution_to_project.rst

```diff
@@ -70,10 +72,10 @@ Examples:
 
 .. _Contributing-to-community:
 
-Contributing to the community
+Contributing to the Community
```

Comment:
Shouldn't this stay lower-case. Also in line 78.

Reply:
maybe:D, OK, I'll change this

Reply:
ah, yeah, it definitely should, thanks, fixed:)